tests/test_table.py

```diff
@@ -41,7 +47,7 @@ def test_get_access_token(self):
     @testmode("workspace_access")
     def test_print_column(self):
         table_name = "simple_germline_variation"
-        column = "file_name"
+        column = "pfb:file_name"
```

Comment:
Similar to the above, should these tests be changed to continue to test the old prefix, for example, using something like `for column in ["pfb:file_name", "file_name"]`?